services/ingestion/regengine_ingestion/parsers/text_parser.py

```
@@ -38,7 +38,7 @@ def parse(self, content: bytes, metadata: Optional[Dict[str, Any]] = None) -> st
         try:
             return content.decode("latin-1")
         except Exception:
-            pass
+            pass  # latin-1 decode should never fail; fall through to ignore-mode
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)